### PR TITLE
put default tasks in a database using restdb service and acquire them…

### PR DESCRIPTION
**Pull request by dvnrsn**
… via fetch

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import React, { useState } from "react";
 import "./styles.css";
 import { Modal } from "./modal/modal";
-import {days as dayKeys, fullDays, defaultTasks} from './helper.js';
```
Comment (dvnrsn) [inline src/App.js:4]: moved defaultTasks to restdb

File: src/App.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+import {days as dayKeys, fullDays} from './helper.js';
 import TaskCard from "./task-card.component";
+import useGetTasks from "./use-get-tasks";
```
Comment (dvnrsn) [inline src/App.js:6]: react custom hook that fetches tasks from restdb

File: src/App.js [hunk 3]
```diff
@@ -19,7 +22,7 @@ export default function App() {
     const all = task.days?.[0] === "all";
     return dayKeys.map(dk => {
       return task.days.some(d => d.day.includes(dk)) || all ? (
-        <TaskCard dk={dk} task={task} toggleDone={toggleDone}/>
+        <TaskCard key={dk} task={task} toggleDone={toggleDone}/>
```
Comment (dvnrsn) [inline src/App.js:25]: this makes react happy

File: src/App.js [hunk 4]
```diff
@@ -43,7 +46,7 @@ export default function App() {
         ))}
       </div>
       {tasks.map(task => (
-        <div key={task.id} className="row">
+        <div key={task._id} className="row">
```
Comment (dvnrsn) [inline src/App.js:49]: task ids from restdb are prepended by underscore

File: src/styles.css [hunk 5]
```diff
@@ -2,6 +2,7 @@
   font-family: sans-serif;
   text-align: center;
   padding: 30px;
+  width: 90%;
```
Comment (dvnrsn) [inline src/styles.css:5]: better styling, might as well fill most the width

File: src/task-card.component.js [hunk 6]
```diff
@@ -1,11 +1,10 @@
-import React, { useEffect, useState } from "react";
```
Comment (dvnrsn) [inline src/task-card.component.js:1]: appease eslint

File: src/App.js (hunk 2 above)
Comment (tcovington1) [reply src/App.js:6]: I'm not very good at custom hooks, something I need to study more.

File: src/App.js (hunk 3 above)
Comment (tcovington1) [reply src/App.js]: Yeah I like that 👍

**Pull request merged**